build/yamls/antrea.yml

```diff
@@ -3495,7 +3495,7 @@ data:
     # stream of packets, a flow record will be exported to the collector once the elapsed
     # time since the last export event is equal to the value of this timeout.
     # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-    #activeFlowExportTimeout: "60s"
+    #activeFlowExportTimeout: "30s"
```

Comment:
maybe worth mentioning the commit message / PR description

Reply:
done